### PR TITLE
Ansuman0 ansuman07 fixess for does this library supports selenium 4.6.0 

### PR DESCRIPTION
**Pull request by Ansuman0**

### CODE REVIEW

**Review by Ansuman0 — COMMENTED** [review]
Verified

File: pom.xml
Comment (sukgu) [inline pom.xml:1]: git diff should show only your changes. Please modify it from a fresh base.

File: igned-tag
Comment (sukgu) [inline igned-tag:1]: We don't need this file.

File: .vscode/settings.json
Comment (sukgu) [inline .vscode/settings.json:1]: pls don't local files in the commit. We should add it in gitignore.

File: src/main/java/io/github/sukgu/Shadow.java
Comment (sukgu) [inline src/main/java/io/github/sukgu/Shadow.java:1]: Only valid changes should be pushed in commit.

File: src/main/java/io/github/sukgu/support/BaseBy.java
Comment (sukgu) [inline src/main/java/io/github/sukgu/support/BaseBy.java:1]: Pls push only valid changes.

**Comment by sukgu** [comment]
All your commits should be verified like the 1st commit, pls check the snapshot below.

image

**Comment by Ansuman0** [comment]
ok
________________________________

**Comment by Ansuman0** [comment]
Hi,
I am  getting this issue .As its private i can not override also .But if i change  to Public  than we can over ride ,Based on your suggestion i will change other wise its not  possible to update to selenium 4.0 above also ?Please do replay I need to change  or not .
[cid:c82208ba-a348-4766-9f3f-1e6fa5b67481]
________________________________

**Comment by DarkWingDuckWing** [comment]
I'm getting the following errors when building locally with `mvn clean install`.
```
[ERROR] Failures: 
[ERROR]   ShadowTest.testGetAllObject:77 
Expected: a value greater than <0>
     but: <0> was equal to <0>
[ERROR] Errors: 
[ERROR]   ShadowTest.testAPICalls1:92 » IndexOutOfBounds Index 0 out of bounds for lengt...
[ERROR]   ShadowTest.testAPICalls2:101 » IndexOutOfBounds Index 0 out of bounds for leng...
[ERROR]   ShadowTest.testAPICalls3:109 » NoSuchElement vt-ui-shell vt-ui-button[data-rou...
[ERROR]   ShadowTest.testAPICalls4:117 » NoSuchElement vt-ui-shell vt-ui-button[data-rou...
[ERROR]   ShadowTest.testAPICalls5:126 » NoSuchElement vt-ui-shell vt-ui-button[data-rou...
[ERROR]   ShadowTest.testHighlightWithDefaultColor:62 » NoSuchElement vt-ui-shell vt-ui-...
[ERROR]   ShadowTest.testHighlightWithGrennColor:56 » NoSuchElement vt-ui-shell vt-ui-bu...
[ERROR]   ShadowTest.testJSInjection:68 » NoSuchElement vt-ui-shell vt-ui-button[data-ro...
[INFO] 
[ERROR] Tests run: 47, Failures: 1, Errors: 8, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  43.211 s
[INFO] Finished at: 2023-10-23T14:59:01-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.0:test (default-test) on project automation: There are test failures.
```

**Pull request closed without merging**